### ISSUE TITLE
Transfer FreeCEN1 Welsh help to FreeCEN2

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeCENMigration#1197]
E2 U3 S8 C5: P18

**Comment by PatReynolds** [comment]
http://freepages.rootsweb.com/~bowbren/genealogy/Cymru__Wales.htm needs review / transfer to FreeCEN2 / update

**Comment by Captainkirkdawson** [comment]
Surely all we need is the Welsh Translations

**Comment by PatReynolds** [comment]
I would think so - http://freepages.rootsweb.com/~bowbren/genealogy/Glamorgan%201871.htm (for example) is well out of date (a quick check suggests so).

**Comment by Captainkirkdawson** [comment]
They are all out of date because they cannot be updated on rootsweb

**Comment by FreecenBren** [comment]
I presume you mean this as below  from my old Roots Web Page.




Welsh
Basic meaning
	Welsh
Basic meaning

Aber
a river mouth or
 confluence of waters
Gwaelod
bottom or foundation

Aelwyd
hearth or home
Gwyn
white or holy

Afon
a river
Gwynt & wynt
wind

Allt
a hill
Hafod
summer dwelling or summer pasture

Awel
a breeze
Haul
the sun

Bach & Fach
small little
Hen
old

Bae
a bay
Hendre
winter dwelling

Betws
a house of prayer
Heulog
sunny

Blaen
the source of a river or 
head of a valley
Hyfryd
pleasant agreeable delightful

Bodlon
satisfied content
Llan
a church or parish

Bron
the breast of a hill
Llyn
a lake or pool

Bryn
a hill
Llys
a hall court  or palace

Buarth
farmyard
Maes
a field

Bwlch
a pass or gap
Mawr & Fawr
big large

Cae
a field
Melin
a mill

Caer & Gaer
a fort
Minffordd
the wayside

Canol
mid or centre
Mor
sea

Capel
a chapel
Morfa
sea marsh or bog

Caredig
kind
Mynydd
a mountain

Carreg
stone
Nant
a stream or brook

Cartref
abode residence
Newydd
new

Castell
a castle
Ochr
beside

Cefn
a ridge
Pant
hollow dent or valley

Celyn
holly
Pen
head top or end

Cloch
a bell
Pentre
a village

Coch & Goch
red
Perllan Berllan
an orchard

Coed
a wood
Pistyll
a waterfall

Craig & Graig
a rock outcrop or crag
Plas
mansion large house

Croes
a cross
Pont
a bridge

Cwm
a valley
Pwll
a pool

Derwen
oak
Rhestr
a row or line

Draw
yonder beyond
Rhos
moorland

Dyffryn
a valley
Rhyd
a ford

Dwr
water
Tan
under beneath

Eglwys
church
Teg
fair fine beautiful

Fan & Ban
summit peak
Terfyn
end or boundary

Ffordd
road
Tirion
gentle gracious

Ffydd
faith confidence or belief
Tre
a homestead or town

Ffynnon
a well
Ty
a house

Glan
river bank or seashore
Tyddyn
a small farm or smallholding

Glyn
a glen or valley
Ucha(f)
highest
 
Please note that I have asked ‘Roots web Free Pages’ to remove the set of indexes for years and had no success.
Other  people are the same including a few Family History Society members.
I you can get them to take them down then good luck!

None of the other pages in the set can be relied on as many Counties have changed a lot in 5 years for FreeCEN.
 
This is the original Welsh  list above is from  Natalia, the Ex COORD for Wales.
 
They are all still on my Front Page so that tells you how old they are.
So yes it would be out of date,  to  be precise Nov 2016 when last updated.


I am sure Kelvin will update what you need. Unless you have someone else  in mind


http://freepages.rootsweb.com/~bowbren/genealogy/Cymru__Wales.htm
 
http://freepages.rootsweb.com/~bowbren/genealogy/index.htm


Brenda

**Comment by PatReynolds** [comment]
I have grabbed the html and saved it here: https://drive.google.com/file/d/15JXDzUvxN3GJehswph8vy_8jZiLzaoAR/view?usp=sharing

It needs:
[a] re-writing to our standards (e.g. colours, fonts).
[b] consultation with volunteers as to what else they have spotted / come across

**Comment by geoffj-FUG** [comment]
Pat
Would it help if I included this info as another Appendix in the uopdate of the FreeCEN2 Manual?
Geoff

**Comment by geoffj-FUG** [comment]
I am happy to add an additional appendix. Please send me the text to look at.

If there is anything that is technically out of date, such as we can now use accented letters in some fields and the change of language codes, then I will add it.

To play devil's advocate - this would appear to be a Gazetteer menu item but out action menu structure does not fit.

Geoff

**Comment by PatReynoldsFUG** [comment]
@PatReynoldsFUG to get in touch with Kelvin.

**Comment by PatReynoldsFUG** [comment]
Moving to Needs Review as I'm unsure what still needs to be done.

**Comment by DeniseColbert** [comment]
Pat to share draft with Kelvin and Chris for them to update and then create a PDF for the website

**Comment by PatReynoldsFUG** [comment]
As noted above I only have a csv file.    I have sent you a link to it, Denise.

**Comment by DeniseColbert** [comment]
@PatReynoldsFUG to turn info on CSV into PDF for website.

**Comment by PatReynoldsFUG** [comment]
I have a note that I am to "look at different versions and choose one" - please provide links to / tell me where these are.

**Comment by DeniseColbert** [comment]
(look in inbox)

**Comment by PatReynoldsFUG** [comment]
Shared 30/4/2025

**Comment by PatReynoldsFUG** [comment]
Pat to send Geoff text re Welsh map sites to add to the Validator's handbook

**Comment by PatReynoldsFUG** [comment]
Pat waiting on Geoff and Kelvin to review text.

**Comment by PatReynoldsFUG** [comment]
I _think_ what was agreed in the meeting was:
Geoff to amend handbook to provide links to maps (for those needing to add a place to the gazzateer and needing to find coordinates.
Pat to add 'finding places in wales' help for researchers.

**Comment by PatReynoldsFUG** [comment]
Wrote to Mapio Cymru (via Facebook) asking if it is possible to get lat and long from their site - that only gives Postcode area.

Wrote first draft for the Validators and sent to Geoff and Kelvin. Began work on the Welsh Placenames Help for Researchers.

**Comment by geoffj-FUG** [comment]
Pat

 

The Gazetteer has latitude and longitude. 100% of entries have their latitude and longitude identified.

 

Lat and long are identified by the validator. Google the place name and county followed by the word coordinates displays the latitude and longitude for that place. There is 0.5% of places that this does not work for. Click on the place on Google maps and it is displayed and can be copied and pasted into the Gazetteer.

 

Geoff

**Comment by PatReynoldsFUG** [comment]
Have updated the draft with some additional instructions about getting lat and long from Mapio Cymry - resent to Geoff and Kelvin 8 May

**Comment by DeniseColbert** [comment]
@geoffj-FUG to review

@PatReynoldsFUG to turn attention to researcher version

**Comment by Vino-S** [comment]
@PatReynoldsFUG to discuss with someone with Welsh knowledge 
@Geoff suggestion: Include Welsh chapman code

**Comment by DeniseColbert** [comment]
@geoffj-FUG to review email from Pat on this

**Comment by geoffj-FUG** [comment]
The welsh names for each of the historical counties has been added to the draft Appendix B together with the applicable Chapman Code.
Geoff

**Comment by PatReynoldsFUG** [comment]
Thank you, Geoff.  Have you added my text?  Also, can you (with Kevin) finalise for public use?

**Comment by DeniseColbert** [comment]
Geoff has added the county names in Welsh to appendix B (CHP codes). Will be released in January in next Handbook update.

**Comment by geoffj-FUG** [comment]
FreeCEN Committee has suggested that Kelvin or another Welsh speaker should be asked to review the content of the welsh help and polish it if required..
Chris Mahoney to consult with Kelvin or an alternative and ask them to join the Committee Meeting in early 2025 to discuss this. Pat will be back by then.
Geoff

**Comment by DeniseColbert** [comment]
- Contact AWFHS for a Welsh speaker to help
- Use their table on ENG/Welsh places/counties (separate story) for Gazetteer quality

**Comment by PatReynoldsFUG** [comment]
Wrote to AWFHS re a new guide to using Censuses returned in Welsh 17 Mar 2025

**Comment by PatReynoldsFUG** [comment]
https://docs.google.com/document/d/1jiaUR3xhPwqxwrLD5xAsQO6WnvlkEBDlC1YwMjeTvYY/edit?usp=sharing has the list of potential topics for inclusion (taken from the other story re Welsh Help).

**Comment by PatReynoldsFUG** [comment]
Contated Wikitree Welsh project managers for advice 29 Apr 2025.

**Comment by PatReynoldsFUG** [comment]
Pat to ask Kelvin.

**Comment by DeniseColbert** [comment]
Kelvin is happy to review Pat's work to suggest improvements.

**Comment by geoffj-FUG** [comment]
@PatReynolds 

Once we have some text I can include it in the handbooks as an Appendix. That will be easier than creating a web page.

Geoff

**Comment by DeniseColbert** [comment]
@philr1950 to take a look at moving this forward

**Comment by philr1950** [comment]
Now that google can translate almost anything - is this still a requirement?

**Comment by geoffj-FUG** [comment]
Phil

 

We have a scrum on Wednesday. We can bring it up then.

 

Geoff

**Comment by DeniseColbert** [comment]
To check with Pat via email

**Comment by DeniseColbert** [comment]
No reply after 5 weeks, closing
